ballsdex/settings.py

```diff
@@ -247,6 +263,7 @@ def update_settings(path: "Path"):
     add_max_attack = "max-attack-bonus" not in content
     add_max_health = "max-health-bonus" not in content
     add_plural_collectible = "plural-collectible-name" not in content
+    add_packages = "packages" not in content
```

Comment:
Won't be detected otherwise as "ballsdex.packages.countryballs.spawn.SpawnManager" already exists in the default config
```suggestion
    add_packages = "packages:" not in content
```